plugins/out_datadog/datadog.c

```diff
@@ -227,8 +245,11 @@ static int datadog_format(struct flb_config *config,
              * (so they won't be packed as attr)
              */
             if (ctx->remap && (ind = dd_attr_need_remapping(k, v)) >=0 ) {
-                remapping[ind].remap_to_tag(remapping[ind].remap_tag_name, v,
-                                            remapped_tags);
+                ret = remapping[ind].remap_to_tag(remapping[ind].remap_tag_name, v,
+                                                  &remapped_tags);
+                if (ret < 0) {
+                    flb_plg_error(ctx->ins, "Failed to remap tag: %s, skipping", remapping[ind].remap_tag_name);
+                }
```

Comment:
If someone from datadog could comment on this bit specifically, that'd be awesome.

So this code adds the ECS task metadata (cluster name, task arn, etc) to the datadog tags. And its very unlikely for this code to fail... that only happens if there is an alloc failure. But, if there is a failure, what should we do?

I was thinking that just skipping and applying the next tag is best. Technically I think the tag string has to be in a nice format like `key:val,key2:val2`. When it fails, we don't know if it failed in the middle of adding a tag, so at the continue here you could have an incomplete string like `key:val,key2:`. I do not know how bad this is. 

My guess was that continuing and risking that the tags are mis-formatted is better than just discarding the tags or discarding the record.